consensus/types/src/beacon_state/tree_hash_cache.rs

```diff
@@ -335,14 +365,6 @@ impl<T: EthSpec> BeaconTreeHashCacheInner<T> {
             hasher.write(leaf.as_bytes())?;
         }
 
-        // Withdrawal indices (Capella and later).
```

Comment:
I moved these because their exclusion from `recalculate_tree_hash_leaves` seems to have been causing issues with merkle proof calculations